cernopendata/jsonschemas/records/record-v1.0.0.json

```diff
@@ -286,6 +286,71 @@
           "description": "A description of the methodology used for the production of this data/software",
           "type": "string"
         },
+        "steps": {
+          "items":[
+            {
+              "properties": {
+                "configuration_files": {
+                  "items": [
+                    {
+                      "properties": {
+                        "script": {
+                          "description": "A cmsdriver script",
+                          "type":"string"
+                        },
+                        "title": {
+                          "description": "Title of the script",
+                          "type":"string"
+                        },
+                        "cms_confdb_id": {
+                          "description": "the cms_confdb_id (applies to CMS records)",
+                          "type":"string"
+                        },
+                        "process": {
+                          "description": "Indicating the process level (e.x. HLT, RECO etc.)",
+                          "type":"string"
+                        },
+                        "recid": {
```

Comment:
"The internal ID of the related record, if it is another Open Data record"

see the description string for `recid` elsewhere in the data model